app/forms/renewal_start_form.rb

```diff
@@ -0,0 +1,41 @@
+class RenewalStartForm
+  include ActiveModel::Model
+
+  attr_accessor :reg_identifier
+
+  def initialize(transient_registration)
+    @transient_registration = transient_registration
+    # Get values from transient registration so form will be pre-filled
+    self.reg_identifier = @transient_registration.reg_identifier
+  end
+
+  validates :reg_identifier, presence: true
+  validate :transient_registration_valid?
+
+  def submit(params)
```

Comment:
Good point. I've chosen to keep the first `params` as `params` as this will be more consistent with the other forms but have renamed the one getting data from the registration.